contents/docs/libraries/ios/index.mdx

```diff
@@ -136,6 +136,58 @@ This will remove the Super Property and subsequent events will not include it.
 
 If you are doing this as part of a user logging out you can instead simply use `PostHogSDK.shared.reset` which takes care of clearing all stored Super Properties and more.
 
+## Amending event properties before they are captured
```

Comment:
I wonder if we could extract this to a file and let the other SDKs add their code snippet with an example, JS already has this https://posthog.com/docs/libraries/js#amending-events-before-they-are-captured so its less dupe content, Android will need it.

Reply:
No need to block this.
I can show you today later on but feel free to merge it.